### DISCUSSION TITLE
Which firmware for LCD?

### DISCUSSION THREAD

**Discussion by NiLMoXiE** [mriscoc/Ender3V2S1#430] [Q&A] [answered]
Hello, I have an Ender 3v2 with the DACAi screen. I have noticed that Creality have issued an update to the firmware but not sure which to try, I know I need the "private" folder but there are more files included in the download. 

After extracting I get:

DWIN_SET - Folder for older DWIN screens
private - Folder for DACAI screens
TJC_SET - No idea what this is for
Firmware.bin - No idea what this is for
FlashBin - No idea what this is for

Anyone got any ideas or can I just put it all on the sd card and it will know which files to use?

Attached the zip file if anyone else wants to have a go!

Thanks

[Ender-3 V2_HW_V4.2.2_SW_V1.0.7_202200811.zip](https://github.com/mriscoc/Ender3V2S1/files/9739811/Ender-3.V2_HW_V4.2.2_SW_V1.0.7_202200811.zip)

**Comment by ghost** [comment, marked as answer]
For the DACAI screen you need the private folder.
Save this folder on the SDCARD and insert it into the slot behind your printer's screen and turn it on, the screen will change color and in a few moments it's ready.
Remember it's the screen sdcard

**Reply by NiLMoXiE** [reply]
Thanks, I just wasn't sure if that was all I needed, any idea what the firmware.bin, FlashBin and TJC_SET is for?

**Reply by ghost** [reply]
> Thanks, I just wasn't sure if that was all I needed, any idea what the firmware.bin, FlashBin and TJC_SET is for?

I was reading the creality firmware txt file and they say the following:
TJC_SET is the firmware when you have touch screen.
The FlashBin and Firmware.bin files must be inserted in the sdcard and inserted in the screen slot after the update with the file in the private folder in the case of DACAI screens

**Reply by NiLMoXiE** [reply]
Thank you - any chance you could link me the firmware txt file??

**Reply by ghost** [reply]
here is the file
[readme.txt](https://github.com/mriscoc/Ender3V2S1/files/9741710/readme.txt)

**Reply by NiLMoXiE** [reply]
Thank You
